inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/BuiltInFieldInfo.java

```diff
@@ -27,6 +27,7 @@
 public class BuiltInFieldInfo extends FieldInfo {
 
     public enum BuiltInField {
+        DATA,
```

Comment:
move this to the end and give it a more specific name such as 'MYSQL_METADATA_DATA'

Reply:
done